core/src/main/java/io/questdb/griffin/model/QueryModel.java

```diff
@@ -1687,6 +1754,16 @@ private void toSink0(CharSink<?> sink, boolean joinSlave, boolean showOrderBy) {
             sink.putAscii(" sample by ");
             sampleBy.toSink(sink);
 
+            if (sampleByFrom != null) {
```

Comment:
Shouldn't we also print fillFrom/fillTo? Also, these sampleByFrom/To and fillFrom/To fields look the same, so it's worth at least adding a clarifying comment somewhere.